contrib/windows/msys_build.sh

```diff
@@ -206,3 +206,4 @@ cat Make.user
 make -j3 VERBOSE=1
 make build-stats
 #make debug
+make prefix=/c/projects/julia/usr VERBOSE=1 -C examples
```

Comment:
When I remove this, it ends up looking for `/c/projects/julia/examples/julia-251f61c05f/bin/julia.exe`. How can I determine where julia was installed from the examples Makefile, or this script?